app/matplotlib.ipynb

```diff
@@ -159,20 +171,23 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
-   "outputs": [],
    "source": [
     "data = np.random.normal(0, 1, 1000)\n",
+    "plt.hist(data, bins=100)\n",
+    "plt.title(\"Random digits\")\n",
     "\n",
-    "# write your code here\n"
+    "plt.xlabel(\"x\")\n",
+    "plt.ylabel(\"y\")"
```

Comment:
You need to add `plt.show()` at the end of this cell to display the histogram as per the task requirements.